src/app/loader/index.test.js

```diff
@@ -22,7 +22,7 @@ const expected = {
   service_B: {
     name: 'Service B',
     documents: {
-      'Terms of Service': {
+      'Privacy Policy': {
         fetch: 'https://www.serviceb.example/tos',
```

Comment:
This is inconsistent with the document type above. I know it's only a test, but it'll make debugging much harder if it ever comes to logging what happens 😉